R/plotCCA.R

```diff
@@ -0,0 +1,504 @@
+#' Plot RDA or CCA object
+#'
+#'
+#' @param object a
+#'   \code{\link[SummarizedExperiment:SummarizedExperiment-class]{SummarizedExperiment}}, a
+#'   \code{\link[TreeSummarizedExperiment:TreeSummarizedExperiment-constructor]{TreeSummarizedExperiment}}
+#'   or a rda/cca object. The latter is the output of \code{\link[mia:runCCA]{calculateRDA}}.
```

Comment:
SingleCellExperiment. You could just state here TreeSUmmarizedExperiment for consistency (and matrix)